+++
title = "Comammox"

# Date first published.
date = "2018-01-25"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Medini K Annavajhala","Vikram Kapoor","Jorge Santo-Domingo","Kartik Chandran"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "*Environmental Science and Technology Letters*"
publication_short = "*Environ Sci Technol Lett*"

# Abstract and optional shortened version.
abstract = ! "Complete ammonia oxidation (comammox) to nitrate by certain Nitrospira-lineage bacteria (CMX) could contribute to overall nitrogen cycling in engineered biological nitrogen removal (BNR) processes in addition to the more well-documented nitrogen transformations by ammonia-oxidizing bacteria (AOB), nitrite-oxidizing bacteria (NOB), and anaerobic ammonia-oxidizing (anammox) bacteria (AMX). A metagenomic survey was conducted to quantify the presence and elucidate the potential functionality of CMX in 16 full-scale BNR configurations treating mainstream or sidestream wastewater. CMX proposed to date were combined with previously published AOB, NOB, and AMX genomes to create an expanded database for alignment of metagenomic reads. CMX-assigned metagenomic reads accounted for between 0.28 and 0.64% of total coding DNA sequences in all BNR configurations. Phylogenetic analysis of key nitrification functional genes *amoA*, encoding the α-subunit of ammonia monooxygenase, *haoB*, encoding the β-subunit of hydroxylamine oxidoreductase, and *nxrB*, encoding the β-subunit of nitrite oxidoreductase, confirmed that each BNR system contained coding regions for production of these enzymes by CMX specifically. Ultimately, the ubiquitous presence of CMX bacteria and metabolic functionality in such diverse system configurations emphasizes the need to translate novel bacterial transformations to engineered biological process interrogation, operation, and design."
abstract_short = ""

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = true
featured = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
projects = ["nutrient-cycling"]

# Links (optional).
url_pdf = "pubs/Annavajhala_ESTLetter_2018.pdf"
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
#url_custom = [{name = "DOI", url = "https://doi.org/10.1021/acs.estlett.7b00577/"}, {name = "Cite", url = "cmx_estltr_cite.bib"}]

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = ""
caption = ""

+++
